test/ICSharpCode.SharpZipLib.Tests/Zip/ZipEncryptionHandling.cs

```diff
@@ -544,6 +550,17 @@ public void CreateZipWithEncryptedEntries(string password, int keySize, Compress
 			}
 		}
 
+		private class NonSeekableMemoryStream : MemoryStream
```

Comment:
trivial, but: there is already a ```MemoryStreamWithoutSeek``` class that does that in the tests?

Reply:
yeah, ` ICSharpCode.SharpZipLib.Tests.TestSupport.MemoryStreamWithoutSeek`